examples/example_container_vulnerabilities_csv.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+from laceworksdk import LaceworkClient
+from datetime import datetime, timedelta
+import csv
+import json
+
```

Comment:
Let's bury the main code under `if __name__ == "__main__":`
```suggestion
if __name__ == "__main__":
```